src/assets/CoffeeScriptAsset.js

```diff
@@ -2,9 +2,13 @@ const JSAsset = require('./JSAsset');
 const localRequire = require('../utils/localRequire');
 
 class CoffeeScriptAsset extends JSAsset {
+  async installParserDependencies() {
+    await localRequire('coffeescript', this.name);
```

Comment:
Seems a little weird to call localRequire to install something. What do you think about having the method return a list of dependencies instead, which would be installed by the base Asset class.

```javascript
async getParserDependencies() {
  return ['coffeescript'];
}
```

You could also still have `installParserDependencies` as a method of the base Asset class which called that method to get a list of deps. And it could be overridden in the case of e.g. Rust where the deps are not npm. WDYT?